app/controllers/authorisedOfficials/AuthorisedOfficialsSummaryController.scala

```diff
@@ -37,16 +37,18 @@ class AuthorisedOfficialsSummaryController @Inject()(
     identify: AuthIdentifierAction,
     getData: UserDataRetrievalAction,
     requireData: DataRequiredAction,
-    view: CheckYourAnswersView,
+    view: OfficialsSummaryView,
     val controllerComponents: MessagesControllerComponents
 )(implicit appConfig: FrontendAppConfig) extends LocalBaseController{
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
 
-    val authorisedOfficialsSummaryHelper = new AddedOfficialsSummaryHelper(Index(0))(request.userAnswers)
+    val firstAuthorisedOfficialsSummaryHelper = new AddedOfficialsSummaryHelper(Index(0))(request.userAnswers)
+    val secondAuthorisedOfficialsSummaryHelper = new AddedOfficialsSummaryHelper(Index(1))(request.userAnswers)
 
-    Ok(view(authorisedOfficialsSummaryHelper.authorisedRows, AuthorisedOfficialsSummaryPage,
-    controllers.authorisedOfficials.routes.AuthorisedOfficialsSummaryController.onSubmit(), h2Required = true))
+    Ok(view(firstAuthorisedOfficialsSummaryHelper.authorisedRows, secondAuthorisedOfficialsSummaryHelper.otherRows,
```

Comment:
just to check, should the secondAuthorisedOfficialsSummaryHelper really call otherRows, and not authorisedRows?

Reply:
yes its required to get rid of additional rows which is there in authorisedRows (applicable for only for 1st authorised official)